### PR TITLE
Add genius X and 10000 and add disclaimer for supported devices

### PR DESCRIPTION
**Pull request by Lash-L**
## Proposed change

some users have reported being confused by the fact that even though their toothbrush isn't on the supported list, it still works. I figured a disclaimer would help limit the amount of people asking in forums or on issues. I saw someone state that they didn't buy an io 10 because they thought it would not be supported, when in reality, we just don't have data for it.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by Lash-L** [comment]
@bdraco what do you think about the disclaimer I added above supported devices, worth it?

File: source/_integrations/oralb.markdown [hunk 1]
```diff
@@ -20,7 +20,9 @@ Integrates [Oral-B](https://oralb.com/) devices into Home Assistant.
 
 {% include integrations/config_flow.md %}
 
-The Oral-B integration will automatically discover devices once the [Bluetooth](/integrations/bluetooth) integration is enabled and functional.
+The Oral-B integration will automatically discover devices once the [Bluetooth](/integrations/bluetooth) integration is enabled and functional. 
+
+All Bluetooth-enabled Oral-B toothbrushes should be discovered, but they may be incorrectly identified or may have some modes missing or incorrectly identified. The following devices have been correctly identifed and should work as expected.
```
Comment (bdraco) [inline source/_integrations/oralb.markdown]:
```suggestion
The integration can discover most Bluetooth-enabled Oral-B toothbrushes. Brushes not listed as supported below may not be correctly identified or have some modes missing.
```

**Comment by bdraco** [comment]
> @bdraco what do you think about the disclaimer I added above supported devices, worth it?

I tweaked the language a bit to be more concise.  Let me know what you think.

**Comment by Lash-L** [comment]
> > @bdraco what do you think about the disclaimer I added above supported devices, worth it?
> 
> I tweaked the language a bit to be more concise. Let me know what you think.

I like those changes. Definitely more concise. I went ahead and committed them.

**Review by bdraco — APPROVED** [review]

**Review by fabaff — APPROVED** [review]

**Pull request merged**